nuget/bblanchon.PDFium.Binaries.macOS.nuspec

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+  <metadata>
+    <id>bblanchon.PDFium.Binaries.macOS</id>
+    <version>$version$</version>
+    <authors>Benoit Blanchon</authors>
+    <owners>Benoit Blanchon</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <license type="expression">Apache-2.0</license>
+    <licenseUrl>https://licenses.nuget.org/Apache-2.0</licenseUrl>
+    <projectUrl>https://github.com/bblanchon/pdfium-binaries</projectUrl>
+    <description>This package contains pre-compiled binaries of the PDFium library, an open-source library for PDF manipulation and rendering.</description>
+    <releaseNotes>$releaseNotes$</releaseNotes>
+    <copyright>Copyright © Benoit Blanchon 2017-2022</copyright>
+    <tags>PDFium PDF binaries library native Chromium</tags>
+    <repository type="git" url="https://github.com/bblanchon/pdfium-binaries.git" branch="$branch$" commit="$commit$" />
+  </metadata>
+
+  <files>
+    <!-- the native PDFium binaries -->
+    <!-- see list of available runtime identifiers -->
+    <!-- https://docs.microsoft.com/en-us/dotnet/core/rid-catalog -->
+    <file src="pdfium-mac-arm64/lib/GNUSparseFile.0/libpdfium.dylib" target="runtimes/osx-arm64/native/libpdfium.dylib" />
```

Comment:
`GNUSparseFile.0`: this is a mistake on my side.

Reply:
Ok, I'll change it to `src="pdfium-mac-arm64/lib/libpdfium.dylib"` then.
I was wondering about this, too. But I thought that is some kind of special Apple ARM64 folder magic.

Reply:
I'm still trying to figure it out because the [build artifact](https://github.com/bblanchon/pdfium-binaries/suites/5117161337/artifacts/153650016) is correct.
I wonder if it could come from `tar`...